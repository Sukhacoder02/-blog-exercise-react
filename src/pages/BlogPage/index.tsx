import * as React from 'react';
import './BlogPage.css';

import { Navbar, Footer, MainDiv } from '../../components';

const BlogPage = () => {
  return (
    <div className="body">
      <Navbar />
      <MainDiv />
      <Footer />
    </div>
  );
};
export default BlogPage;
