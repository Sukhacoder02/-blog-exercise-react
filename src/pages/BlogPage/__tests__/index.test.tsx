import * as React from 'react';
import { render } from '@testing-library/react';
import BlogPage from '..';
jest.mock('react-router-dom');
describe('Page BlogPage', () => {
  it('Should render without crashing', () => {
    const { asFragment } = render(<BlogPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
