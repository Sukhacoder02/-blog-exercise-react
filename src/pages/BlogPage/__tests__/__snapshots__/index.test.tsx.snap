// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page BlogPage Should render without crashing 1`] = `
<DocumentFragment>
  <div
    class="body"
  >
    <div>
      <header>
        <div
          class="heading"
        >
          <h2>
            The Artifact
          </h2>
          <h4>
            <em>
               Culture 
            </em>
            & 
            <em>
              Art
            </em>
             Blog
          </h4>
        </div>
      </header>
      <div
        class="navBar"
      >
        <nav>
          <a
            href="#"
          >
            Blog
          </a>
          <a
            href="#"
          >
            About
          </a>
          <a
            class="contactLink"
            href="#"
          >
            Contact
          </a>
        </nav>
      </div>
    </div>
    <div
      id="main"
    >
      <div
        class="cards"
      />
    </div>
    <footer>
      <div
        class="footerHeading"
      >
        <h4>
          @artifact.com 2023
        </h4>
      </div>
    </footer>
  </div>
</DocumentFragment>
`;
